### PR TITLE
Update docs for amcrest component after small fix request

### PR DESCRIPTION
**Pull request by blademckain**
## Proposed change
Update of the documentation related to[ the fix request](https://github.com/home-assistant/core/pull/40496) and addition of features for the amcrest component

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: [here](https://github.com/home-assistant/core/pull/40496)
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/amcrest.markdown [hunk 1]
```diff
@@ -135,6 +137,14 @@ control_light:
   required: false
   type: boolean
   default: true
+event:
```
Comment (frenck) [inline source/_integrations/amcrest.markdown]: We no longer accept YAML configuration for integrations that connect to devices or services.

Instead, a configuration flow via the UI should be used.

For more information about this decision see:

- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

To get started on the integration configuration, see our developer documentation:

<https://developers.home-assistant.io/docs/config_entries_config_flow_handler>

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by blademckain** [comment]
hello @frenck 
I understand the decision that has been made to force the use of config flow. But I am not the maintainer of the amcrest component. I just made a few small changes to make the component compatible with the new version of the python\amcrest library. I ask you if possible to approve these changes, then I will contact the maintainer of the amcrest component to make the conflig flow together. There are currently [many people waiting for changes](https://github.com/tchellomello/python-amcrest/issues/137) to this component in order to make their devices work.

**Comment by frenck** [comment]
@blademckain We accept changes, but don't allow changes to the YAML configuration schema, as per ADR-0010.

File: source/_integrations/amcrest.markdown (hunk 1 above)
Comment (blademckain) [reply source/_integrations/amcrest.markdown]: added support for config flow as requested

**Review by blademckain — COMMENTED** [review]
done as requested

File: source/_integrations/amcrest.markdown [hunk 2]
```diff
@@ -20,20 +22,18 @@ There is currently support for the following device types within Home Assistant:
...
-amcrest:
-  - host: IP_ADDRESS_CAMERA
-    username: YOUR_USERNAME
-    password: YOUR_PASSWORD
-
-```
+<div class='note'>
+  The  amcrest integration reads the previous configurations into the configuration.yaml file.
+It is recommended that you remove these configurations after import.
+</div>
```
Comment (frenck) [inline source/_integrations/amcrest.markdown:35]: This is part of the breaking changes sections in the release notes, not or documentation.

Please remove this part.

```suggestion
```

File: source/_integrations/amcrest.markdown [hunk 3]
```diff
@@ -20,20 +22,18 @@ There is currently support for the following device types within Home Assistant:
 - Binary Sensor
 - Camera
 - Sensor
+- Events
 
 ## Configuration
 
-To enable your camera in your installation, add the following to your `configuration.yaml` file:
+For configuration go to the `Integrations pane` on your Home Assistant instance.
```
Comment (frenck) [inline source/_integrations/amcrest.markdown:29]: - We don't use the term `pane`. Please use `panel` instead.
- No need to mention `on your Home Assistant instance`, that is assumed.
- Provide a click path / more extensive instruction to the user on where to click / what to do.

File: source/_integrations/amcrest.markdown [hunk 4]
```diff
@@ -20,20 +22,18 @@ There is currently support for the following device types within Home Assistant:
...
-    password: YOUR_PASSWORD
-
-```
+<div class='note'>
+  The  amcrest integration reads the previous configurations into the configuration.yaml file.
+It is recommended that you remove these configurations after import.
+</div>
 
+Parameters configurable through the `Integrations pane`:
 {% configuration %}
```
Comment (frenck) [inline source/_integrations/amcrest.markdown:37]: This configuration block is for YAML configuration and should be removed.

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**